static/.htaccess

```diff
@@ -10,7 +10,7 @@ Options -Indexes -MultiViews
 
 # 8.0 apis/clients to apis/tools change
 RewriteRule ^docs/8.0/apis-clients(.*)$ /docs/8.0/apis-tools$1 [R=301,L]
-RewriteRule ^optimize/3.8.0/apis-clients/(.*)$ /optimize/3.8.0/apis-tools$1 [R=301,L]
+RewriteRule ^optimize/3.8.0/apis-clients(.*)$ /optimize/3.8.0/apis-tools$1 [R=301,L]
```

Comment:
@christinaausley the "match" on the left side of the rule included a `/` after apis-clients; but the target did not include the `/`. So a URL like `/optimize/3.8.0/apis-clients/optimize-api` was being redirected to `/optimize/3.8.0/apis-toolsoptimize-api/`, causing the broken links.